better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java

```diff
@@ -34,17 +35,17 @@
 
     private OnLinkClickListener onLinkClickListener;
     private final RectF touchedLineBounds = new RectF();
-    private boolean isUrlHighlighted;
+    private boolean isURLHighlighted;
```

Comment:
I would revert all these uppercasing changes. Acronyms should be cased like words. See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case